test/powershell/Modules/Microsoft.PowerShell.Utility/Get-PSCallStack.Tests.ps1

```diff
@@ -79,11 +82,15 @@ Describe "Get-PSCallStack DRT Unit Tests" -Tags "CI" {
         $results[0].Command | Should -Be $scriptFileName
         $results[0].ScriptName | Should -Be $scriptFilePath
         $results[0].ScriptLineNumber | Should -Be 3
-        $results[0].InvocationInfo.ScriptLineNumber | Should -Be 77
+        $results[0].InvocationInfo.ScriptLineNumber | Should -Be 80
 
         $results[1].Command | Should -Be $scriptFileName
         $results[1].ScriptName | Should -Be $scriptFilePath
         $results[1].ScriptLineNumber | Should -Be 7
-        $results[1].InvocationInfo.ScriptLineNumber | Should -Be 77
+        $results[1].InvocationInfo.ScriptLineNumber | Should -Be 80
+    }
+
+    It "Get-PSCallStack returns Arguments" {
+        & { (Get-PSCallStack)[0].Arguments } 'foo'  | Should -Match 'foo'
```

Comment:
Please also add a test for the case where the scriptblock has formal parameters:
```PowerShell
 & { param ($x)  (Get-PSCallStack)[0].Arguments } 'foo' 
```
and another where multiple arguments are passed to the scriptblock:
```powershell
 & { (Get-PSCallStack)[0].Arguments } 'foo'  'bar'
```

Reply:
Done.